sentry_auth_github/constants.py

```diff
@@ -5,6 +5,8 @@
 CLIENT_ID = getattr(settings, 'GITHUB_APP_ID', None)
 
 CLIENT_SECRET = getattr(settings, 'GITHUB_API_SECRET', None)
+if not CLIENT_SECRET:
```

Comment:
Gah, it happened again.
It should `getattr` from `GITHUB_APP_SECRET` instead. I'll fix it in a sec